src/plugins/Responsive.ts

```diff
@@ -6,7 +6,7 @@ import GenericPlugin from './GenericPlugin';
 class ResponsivePlugin extends GenericPlugin {
   public currentConfines: ICurrentConfines = {};
   private EVENT_KEY: string = 'resize';
-  private THROTTLE_DURATION: number = 200;
+  private THROTTLE_DURATION: number = 250;
```

Comment:
should we make this configurable?

Reply:
We could definitely pass that in with configuration.
If you want the feature let me know and i can add it to my list i am knocking out.

Reply:
Lets see what Stephen thinks